avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -682,27 +705,48 @@ def updatePositionC(cfg, particles, dem, force, DT):
   log.debug('Entrained DFA mass: %s kg', np.asarray(massEntrained))
   particles['kineticEne'] = TotkinEneNew
   particles['potentialEne'] = TotpotEneNew
-  if peakKinEne < TotkinEneNew:
-    particles['peakKinEne'] = TotkinEneNew
-  if particles['peakMassFlowing'] < massFlowing:
-    particles['peakMassFlowing'] = massFlowing
-
-  if cfg['stopCritType'] == 'massFlowing':
-    value = massFlowing
-    peakValue = particles['peakMassFlowing']
-    stop = (value <= stopCrit*peakValue) and peakValue > 0
-  elif cfg['stopCritType'] == 'kinEnergy':
-    value = TotkinEneNew
-    peakValue = particles['peakKinEne']
-    stop = value <= stopCrit*peakValue
+
+  if typeStop == 1:
```

Comment:
same here: quick comment about the purpose

Reply:
done